src/app/asign-family/asign-family.component.ts

```diff
@@ -113,30 +114,34 @@ export class AsignFamilyComponent implements OnInit, OnDestroy {
   }
   async editHelper() {
     await this.familyLists.helper.displayEditDialog(this.dialog)
-    if (this.phone != this.familyLists.helper.phone.thePhone)
-      this.phone = this.familyLists.helper.phone.thePhone
+    if (this.identifier != this.familyLists.helper.phone.thePhone)
+      this.identifier = this.familyLists.helper.phone.thePhone
   }
-  async searchPhone() {
+  async searchIdentifier() {
     this.clearHelperInfo(false)
-    let cleanPhone = Phone.fixPhoneInput(this.phone)
+    const cleanPhone = Phone.fixPhoneInput(this.identifier)
+    const identifierType = getIdentifierType(cleanPhone)
 
-    if (this.isValidPhone()) {
-      this.phone = cleanPhone
-      let thisPhone = new Phone(this.phone)
+    if (identifierType === 'phone' && this.isValidPhone()) {
+      this.identifier = cleanPhone
+      let thisPhone = new Phone(this.identifier)
       await this.busy.donotWait(async () => {
         let helper = await remult.repo(Helpers).findFirst({ phone: thisPhone })
         if (helper) {
           this.initHelper(helper)
-        } else if (this.phone == cleanPhone) {
+        } else if (this.identifier == cleanPhone) {
           helper = remult.repo(Helpers).create()
           helper.phone = thisPhone
           this.initHelper(helper)
         }
       })
+    } else {
+      this.identifier = identifierType === 'phone' ? cleanPhone : this.identifier
+      this.helperSuggestions = await searchHelpersByIdentifier(this.identifier)
```

Comment:
I think that this could benefit also from the `donotWait` as defined on line 128

By default in this app any query that takes ~300ms will open a loading ui over the screen - that'll steal the focus from the input.
So for places where we type with autocomplete we use the `donotWait` call to disable that loading ui

Reply:
Sound great